packages/linear-genome-view/src/LinearGenomeView/components/TracksContainer.tsx

```diff
@@ -33,33 +33,37 @@ function TracksContainer({
   model: LGV
 }) {
   const classes = useStyles()
-  const [scheduled, setScheduled] = useState(false)
-  const [delta, setDelta] = useState(0)
+  // refs are to store these variables to avoid repeated rerenders associated with useState/setState
+  const delta = useRef(0)
+  const scheduled = useRef(false)
+
   const [mouseDragging, setMouseDragging] = useState(false)
-  const [prevX, setPrevX] = useState()
+  const prevX = useRef<number | null>(null)
+  useState()
```

Comment:
Is this `useState` a leftover that should have been deleted?

Reply:
Ya, deleted that